python/cudf/cudf/core/column/categorical.py

```diff
@@ -1000,7 +1000,7 @@ def clip(self, lo: ScalarLike, hi: ScalarLike) -> "column.ColumnBase":
     def data_array_view(self) -> cuda.devicearray.DeviceNDArray:
         return self.codes.data_array_view
 
-    def unique(self) -> CategoricalColumn:
+    def unique(self, *args, **kwargs) -> CategoricalColumn:
```

Comment:
Can you explain why this is required? For all other similar changes below too.

Reply:
This was for mypy, but actually now I'm thinking that there is a more precise solution that doesn't involve forcing args and kwargs through everywhere. Let me try it and get back to you.

Reply:
OK these should be fixed now and are no longer necessary.